docs/build/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Building dApps on Celo
+description: A guide  for building on Celo.
+---
+import ColoredText from '/src/components/ColoredText';
+
+Whether you're building your first dApp or looking to integrate an existing protocol onto Celo, we have all the resources and tools you need to get started.
+
+---
+
+## Why Build on Celo?
+
+- **EVM Compatibile:** Celo is fully EVM-compatible, offering the same development experience as Ethereum with improved scalability and lower costs.
+- **Mobile First:** Celo's ultralight client and phone number mapping allows developers to build dApps that enable easy payments via phone numbers, making blockchain accessible to mobile users, even in areas with limited connectivity.
```

Comment:
To be checked that the "ultralight client" is still a thing - is this Plumo?

Reply:
@viral-sangani can you confirm

Reply:
@sodofi , the ultra-light client information should be removed.